yb-voyager/src/metadb/migrationStatus.go

```diff
@@ -10,30 +10,31 @@ import (
 )
 
 type MigrationStatusRecord struct {
-	MigrationUUID                              string            `json:"MigrationUUID"`
-	ExportType                                 string            `json:"ExportType"`
-	ArchivingEnabled                           bool              `json:"ArchivingEnabled"`
-	FallForwardEnabled                         bool              `json:"FallForwardEnabled"`
-	FallbackEnabled                            bool              `json:"FallbackEnabled"`
-	TargetDBConf                               *tgtdb.TargetConf `json:"TargetDBConf"`
-	FallForwardDBConf                          *tgtdb.TargetConf `json:"FallForwardDBConf"`
-	SourceDBAsTargetConf                       *tgtdb.TargetConf `json:"SourceDBAsTargetConf"`
-	TableListExportedFromSource                []string          `json:"TableListExportedFromSource"`
-	SourceDBConf                               *srcdb.Source     `json:"SourceDBConf"`
-	CutoverRequested                           bool              `json:"CutoverRequested"`
-	CutoverProcessedBySourceExporter           bool              `json:"CutoverProcessedBySourceExporter"`
-	CutoverProcessedByTargetImporter           bool              `json:"CutoverProcessedByTargetImporter"`
-	FallForwardSyncStarted                     bool              `json:"FallForwardSyncStarted"`
-	FallForwardSwitchRequested                 bool              `json:"FallForwardSwitchRequested"`
-	FallForwardSwitchProcessedByTargetExporter bool              `json:"FallForwardSwitchProcessedByTargetExporter"`
-	FallForwardSwitchProcessedByFFImporter     bool              `json:"FallForwardSwitchProcessedByFFImporter"`
-	FallBackSyncStarted                        bool              `json:"FallBackSyncStarted"`
-	FallBackSwitchRequested                    bool              `json:"FallBackSwitchRequested"`
-	FallBackSwitchProcessedByTargetExporter    bool              `json:"FallBackSwitchProcessedByTargetExporter"`
-	FallBackSwitchProcessedByFBImporter        bool              `json:"FallBackSwitchProcessedByFFImporter"`
-	ExportSchemaDone                           bool              `json:"ExportSchemaDone"`
-	ExportDataDone                             bool              `json:"ExportDataDone"`
-	YBCDCStreamID                              string            `json:"YBCDCStreamID"`
+	MigrationUUID string `json:"MigrationUUID"`
+	ExportType    string `json:"ExportType"`
+	// FallForwarDBExists                         bool              `json:"FallForwarDBExists"`
+	ArchivingEnabled                                bool              `json:"ArchivingEnabled"`
+	FallForwardEnabled                              bool              `json:"FallForwardEnabled"`
```

Comment:
are we going to rename this also?

Reply:
I consciously did not. My thought process is:
Rename everything related to 
- setup/synchronize
- fallforwardDB->sourceReplicaDB
- fallforward/back switchover -> cutover to *

It still makes sense to have the notion of the fall-forward/fall-back "workflows" as such. 
In this example, because we are importing to source-replica, the "fallforward workflow" is enabled. Another example is that during cutover we have a flag "--prepare-for-fall-back". 

So we still have that terminology, but just in the context of the workflows; and not in the context of commands/DBs. Makes sense?